xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-default/src/test/java/org/xwiki/mentions/internal/DefaultMentionsDataConsumerTest.java

```diff
@@ -320,6 +320,150 @@ void consumeUpdate() throws Exception
                 newCommentXDOM));
     }
 
+    @Test
+    void consumeUpdateBaseObjectNull() throws Exception
```

Comment:
Those tests are difficult to read: couldn't you factorize some piece of code for the fixture of the test? I think most of the content of the test is similar from one test to another no?